charts/flink-job/templates/operator-rolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+# This role binding allows oncall engineers to patch configmaps.
+kind: RoleBinding
+metadata:
+  name: {{ .Release.Name }}-operator-role-binding
+  namespace: {{ .Release.Namespace }}
+subjects:
+- kind: Group
```

Comment:
Let's parameterize this and leave the subjects out of this codebase entirely:

```yaml
# values.yaml
operatorGroups: [system:masters]
```


```yaml
subjects:
{{- range $group := .Values.operatorGroups }}
- kind: Group
   name: {{ $group }}
   namespace: {{ .Release.Namespace }}
{{- end }}
```

This default uses a standard core K8S group (system:masters), but then lets us override it at the chart leve.